### DISCUSSION TITLE
Revert Acceleration behavior (no shuffle on empty deck)

### DISCUSSION THREAD

**Discussion by boshenw** [scarf005/Marisa#146] [Balance]
The original Acceleration behavior would draw cards until the deck becomes empty, then do nothing. This behavior was changed #63 so that it will shuffle the deck when it becomes empty, like any other card draw card. I think the card should go back to its original behavior, for a couple reasons:

1. The card wording supports the original behavior. It does not say "Draw 2" like other draw cards do. Instead it has an alternate wording of "Draw top 2 cards _in_ your draw pile". So, if there's nothing in the draw pile, you do nothing, because there are no cards in it.

2. With a free Amplify effect, Acceleration becomes a 0 cost Skim (with current behavior), which is excellent value and also makes comboing much easier than usual. And Acceleration is a common, so it's not too hard to load up on multiple copies of it.

3. Even without free Amplify you can still combo with similar logic as 2x Flash of Steel or 2x Finesse. You just need a 0 cost attack (like the starter one) to play in between Accelerations. 

The original behavior prevented comboing with just Accelerations because you needed another card to draw "normally" once the deck becomes empty, to shuffle the discard back in. This made comboing trickier because you not only needed another card drawer, you also probably needed extra energy to pay for it. When I first encountered that limitation I thought it was really interesting design, and I believe it was a deliberate way to keep the card's power level balanced.

**Comment by scarf005** [comment]
i see the reasoning behind the suggestion. 

however, marisa already has poor defense and infinite(or many) shuffles using acceleration was a way to prevent getting attacked.

therefore, i'm concerned that reverting acceleration's behaviour would reduce marisa's choice for defenses. 

could you provide a possible solution?

**Comment by boshenw** [comment]
I do agree that her defense sometimes feels a little weak, but I don't think it's so bad a change like this will ruin it. Acceleration can still be used to fill up your hand most of the time. Only when the deck is empty or nearly empty will Acceleration become less useful. At that point you would need to use a card like Milky Way or Gravity Beat (commons) to draw normally to force a shuffle. Although that won't always be possible, the point is that there are options to work around Acceleration's restriction.

I think that the potential upside of the current Acceleration easily enabling infinites on its own (with the right setup) is too high, compared to the potential downside of the old one of sometimes not being playable due to an empty deck.
